packages/encryption/src/lib/encryption.ts

```diff
@@ -383,7 +390,10 @@ export const encryptFileAndZipWithMetadata = async ({
     },
   });
 
-  if (!paramsIsSafe) return;
+  if (!paramsIsSafe) return throwError({
+    message: `authSig, accessControlConditions, evmContractConditions, solRpcConditions, unifiedAccessControlConditions, chain, file, litNodeClient, and readme must be provided`,
```

Comment:
It's a better dev experience if we rather throw a more specific error- which param is unsafe

Reply:
We can add this in a separate PR